kotlinx-coroutines-core/common/src/flow/internal/AbstractSharedFlow.kt

```diff
@@ -31,11 +78,13 @@ internal abstract class AbstractSharedFlow<S : AbstractSharedFlowSlot<*>> : Sync
             }
         }
 
+    /** Allocate a new implementation-representation of a collector, but do not register it anywhere yet. */
```

Comment:
I understand the function meaning but I can't parse "implementation-representation"